flit/vcs/fsl.py

```diff
@@ -0,0 +1,32 @@
+import os
+from subprocess import check_output
+
+name = 'fossil'
+
+def find_repo_root(directory):
+    for p in [directory] + list(directory.parents):
+        if ((p / '.fslckout').is_file()
+            or (p / '_FOSSIL_').is_file()):
+            return p
+
+def _repo_paths_to_directory_paths(paths, directory):
+    # 'fossil ls' gives paths from repo root, which may not be our directory.
```

Comment:
It may need to add the
```python
directory = directory.resolve()
```
that was included in `hg.py`, just to be consistent, but other than that, this still works.